rabbitmq/json-nodes.yml

```diff
@@ -0,0 +1,71 @@
+type: json
+data:
+  url: /api/nodes
+  server: ${SPM_MONITOR_RABBITMQ_API_URL}
+  basicHttpAuthUsername: ${SPM_MONITOR_RABBITMQ_BASICAUTH_USERNAME}
+  basicHttpAuthPassword: ${SPM_MONITOR_RABBITMQ_BASICAUTH_PASSWORD}
+
+observation:
+  - name: nodes
+    metricNamespace: rabbitmq
+    path: $.[?(@.name=${name})]
+
+    metric:
+      - name: file.used
+        source: fd_used
+        type: long_gauge
+        label: used file
+        description: Used file descriptors
+
+      - name: disk.free
+        source: disk_free
+        type: long_gauge
+        label: free space
+        unit: byte
+        description: Current free disk space
+
+      - name: mem.used
+        source: mem_used
+        type: long_gauge
+        label: used memory
+        description: Memory used in bytes
+
+      - name: process.wait
+        source: run_queue
+        type: long_gauge
+        label: warting process
+        description: Average number of Erlang processes waiting to run
```

Comment:
Average? Or current?

Reply:
This is average.
https://pulse.mozilla.org/doc/stats.html
run_queue | Average number of Erlang processes waiting to run.


"

Reply:
Hm, another strange choice, wont be very useful metric, but better than nothing.